index.html

```diff
@@ -209,6 +209,29 @@ <h2>Methods</h2>
       and its privacy and security properties for comparison to other methods.
     </p>
 
+    <section class="informative">
+      <h3>Method Reuse Encouraged</h3>
+      <p>
+        While decentralization is desired in many ways, some convergence upon
```

Comment:
```suggestion
        While many forms of decentralization are desired, some convergence upon
```

Reply:
Applied in 9d682cb2fc686a1e89882525394f305d47f78853